website/src/releases/2022-07/extending-private-installations.mdx

```diff
@@ -0,0 +1,19 @@
+---
+date: 2022-07-20
+title: Installing Custom Applications in multiple Organizations
+description: It is now possible to install a Custom Application in other Organizations where the user has administration rights
+type: announcement
+topics:
+  - Community
+  - Merchant Center
+---
+
+With the [release of Org-level Custom Applications](/releases/2022-02-02-custom-applications-v21) one of the main long-term features with this new setup is to be able to configure a Custom Application once and install it _everywhere_.
```

Comment:
With the [release of Org-level Custom Applications](/releases/2022-02-02-custom-applications-v21), one of the long-term goals for this feature was to enable Custom Applications to be configured only once and installed _everywhere_ needed.
```suggestion
With the [release of Org-level Custom Applications](/releases/2022-02-02-custom-applications-v21) one of the main long-term features with this new setup is to be able to configure a Custom Application once and install it _everywhere_.
```